### PR TITLE
FIX duplicate teleport target overlays

### PR DESCRIPTION
**Pull request by thoys**
When spamming the teleporting buttons I figured out that it created overlays, and didn't remove the older ones.

## Test plan

- Hold teleport button on your controller (center pad on vive controller)
- Spam the trigger button (to teleport), and button mash on the center pad.
- Be sure that the teleport target overlay is gone after releasing all buttons
- Try for both hands

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 11b461a73033867d00763d4c60f7532adae0aba4 available at: https://gist.github.com/e2bd81bc9a462bc019ade8039963e55e<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8271-11b461a.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8271-11b461a.dmg

**Comment by thoys** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: 11b461a73033867d00763d4c60f7532adae0aba4 available at: https://gist.github.com/e2bd81bc9a462bc019ade8039963e55e<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8271-11b461a.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8271-11b461a.dmg

**Comment by thoys** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: 11b461a73033867d00763d4c60f7532adae0aba4 available at: https://gist.github.com/e2bd81bc9a462bc019ade8039963e55e<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8271-11b461a.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8271-11b461a.dmg

**Comment by thoys** [comment]
Come on Gustavo you don't test JavaScript, we all know that! :wink:

**Comment by murillodigital** [comment]
test this please

File: scripts/system/controllers/teleport.js [hunk 1]
```diff
@@ -88,6 +88,7 @@ function Teleporter() {
 
     this.createTargetOverlay = function() {
 
+        _this.deleteTargetOverlay();
```
Comment (huffman) [inline scripts/system/controllers/teleport.js]: is there a reason to recreate the target overlay?

Comment (huffman) [inline scripts/system/controllers/teleport.js]: * versus just returning here if it has already been created

Comment (thoys) [inline scripts/system/controllers/teleport.js]: Sorry was a bit late and wanted to make a `QUICK` fix :)

Comment (thoys) [inline scripts/system/controllers/teleport.js]: How else are we ever going to overflow the Overlay Index ;)

**Comment by hifi-gustavo** [comment]
Build console output for commit: 2adff24a058ab596bae5b427d55b03e72caaf99e available at: https://gist.github.com/e2bd81bc9a462bc019ade8039963e55e<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8271-2adff24.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8271-2adff24.dmg

**Comment by kevinmthomasse** [comment]
testing

**Comment by imgntn** [comment]
i'm still able to get this into a weird state on my hydras; looking for a solid repro path

**Comment by kevinmthomasse** [comment]
Unable to connect to any domain:  Every domain says "Protocol version mismatch - Domain Version :219"

Failure

**Comment by ZappoMan** [comment]
@thoys - I think you need to update from master/rebase... so your PR will be able to connect to the dev domains.

**Comment by thoys** [comment]
please retest after its build @kevinmthomasse thanks :+1:

**Comment by thoys** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: be0ccad0d35db4d2780b1eebf98aa8ba3bad7d0a available at: https://gist.github.com/e2bd81bc9a462bc019ade8039963e55e<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8271-be0ccad.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8271-be0ccad.dmg

**Comment by kevinmthomasse** [comment]
## QA Re-Test
**QA Tester**: KevinMThomas
**QA Pass**: **QA Pass** Success
**Additional Test Results + Observations**:  Held teleport button on your controller on Hydra (number 1) and spammed the trigger button to teleport and the teleport target overlay disappears after releasing all buttons and it works on both hands.

**Pull request merged**